packages/core/docs/recipes/testing.md

```diff
@@ -0,0 +1,198 @@
+
+```yaml
+#./test/docker-compose.yml
+version: '2'
+
+services:
+  tester:
+    expose:
+      - "3000"
+    build:
+      context: ../
+      dockerfile: Dockerfile.test
+    working_dir: /src
+    volumes:
+      - ${PWD}:/src
+    environment:
+      NODE_ENV: "test"
+      DEBUG: ${DEBUG}
+    command: tail -f /dev/null
+```
+
+After performing these steps, you'll be able to run your tests in Docker using the command:
+
+```console
+$: docker-compose -f docker-compose.yml run --rm tester yarn test
+```
+
+The result is going to look like:
+
+```console
+$: docker-compose -f test/docker-compose.yml run --rm tester yarn test
+WARNING: The DEBUG variable is not set. Defaulting to a blank string.
+yarn run v1.19.1
+$ mocha
+
+
+  server
+    ✓ should be able to start (426ms)
+    ✓ should say hello world (160ms)
+
+  2 passing (592ms)
+
+Done in 1.10s.
+```
+
+HOORAY! Your tests run in Docker.
+
+## 4. Update your package.json
+Now you can save the previous command in your `package.json`.
+```json
+{
+  "scripts": {
+    "test:docker": "docker-compose -f test/docker-compose.yml run --rm tester yarn test"
+  },
+}
+```
+After this, you will be able to run your tests using the `yarn test:docker` command.
+
+# `mdep` As a Swedish Knife
+`Microfleet` framework provides a special tool for instrumenting your development process. Please read [this page](https://www.npmjs.com/package/@makeomatic/deploy) to find advanced information and CLI commands.
+
+## 1. Install `mdep` binary
+First of all, you need to install the `@makeomatic/deploy` package, which provides `mdep` CLI command.
+
+```console
+u:~$ npm install --save @makeomatic/deploy
+u:~$ npm install --save-dev cross-env
+u:~$ npm install --save-dev nyc
+u:~$ npm install --save-dev @babel/core @babel/register babel-plugin-istanbul babel-plugin-transform-strict-mode
+```
+
+The package adds `husky` and `semantic-release` packages into your `package.json` and adds Git Commit message validation hook.
+
+Now you can check that installation is successful by running:
+```console
+u:~$ npx mdep --help
+mdep <command>
+
+Commands:
+  mdep docker          manages docker lifecycle
+  mdep test <command>  performs tests in docker
+
+Options:
+  --node, -n                  node version to use when building
+                                                            [default: "12.13.0"]
+  --env, -E                   node environment to build for
+                                                         [default: "production"]
+  --project, -p               project name where this is used
+                                                [default: "microfleet-demo-app"]
+  --docker_build_args, --dba  docker build args
+  --repository, --repo        docker repository to use   [default: "makeomatic"]
+  --version, -v               version of the project to build [default: "1.0.0"]
+  --pkg                       package json path
+  [default: "/home/pajgo/Desktop/makeomatic/microfleet/create-microfleet-app/pac
```

Comment:
depersonalize the path please (just in terms of consistency: we've decided to hide such artifacts few pull requests ago)